ocs_ci/ocs/bucket_utils.py

```
@@ -3618,3 +3618,100 @@ def get_bucket_status_value(mcg_obj, bucket_name, key):
+    for tag_set in TimeoutSampler(
+        timeout=timeout,
+        sleep=sleep,
+        func=get_bucket_tagging,
+        s3_client=s3_client,
+        bucket_name=bucket_name,
+    ):
+        bucket_tags = tag_set_to_dict(tag_set)
+        if all(bucket_tags.get(key) == value for key, value in expected_labels.items()):
+            logger.info(
+                f"Bucket {bucket_name} tags {bucket_tags} match expected labels "
+                f"{expected_labels}"
```

Ruff (0.15.14)

[warning] Logging statement uses f-string

(G004)